compliance_checker/cf/cf_1_6.py

```diff
@@ -828,7 +827,7 @@ def _check_valid_cf_units(self, ds, variable_name):
         # being expressed as "s"/seconds
         if standard_name not in {"time", "forecast_reference_time"}:
             valid_units.assert_true(
-                units_conv.is_convertible(Unit(reference)),
+                units_conv.is_convertible_to(cfutil._units(reference)),
```

Comment:
@pelson  `cf_units` could take a unit string or another Unit instance instance as an argument here. Not a big deal but something to keep in mind if you are aiming for 100% compatibility.

Reply:
Unfortunately this won't be possible - you need a unit system to define a unit, and the unit itself doesn't hold on to the unit system from which it came. 

Created https://github.com/pelson/pyudunits2/issues/7 to track this.

Reply:
Go it. Thanks! It is not a big deal here though, just wanted to keep the original code as close as possible to what we have for cf-units b/c I want to try to support both (cf-units as a "fast" option).